### PR TITLE
PartitionsCacheRepository now logs all its major st…

### PR DESCRIPTION
**Pull request by sergii-vostrikov**
…eps.

Resolves: OLPEDGE-528

Signed-off-by: Sergii Vostrikov <ext-sergii.vostrikov@here.com>

### CODE REVIEW

File: olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp [hunk 1]
```diff
@@ -33,16 +34,18 @@
 // clang-format on
 
 namespace {
+constexpr auto kLogtag = "PartitionsCacheRepository";
```
Comment (andescu) [inline olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp]:
```suggestion
constexpr auto kLogTag = "PartitionsCacheRepository";
```

File: olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp [hunk 2]
```diff
@@ -108,11 +115,11 @@ model::Partitions PartitionsCacheRepository::Get(
 boost::optional<model::Partitions> PartitionsCacheRepository::Get(
     const PartitionsRequest& request) {
   std::string hrn(hrn_.ToCatalogHRNString());
-  auto cachedIds = cache_->Get(
-      CreateKey(hrn, request.GetLayerId(), request.GetVersion()),
-      [](const std::string& serializedIds) {
-        return parser::parse<std::vector<std::string>>(serializedIds);
-      });
+  auto key = CreateKey(hrn, request.GetLayerId(), request.GetVersion());
+  LOG_TRACE_F(kLogtag, "Get '%s'", key.c_str());
```
Comment (andescu) [inline olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp]: LOG_INFO_F?

File: olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp [hunk 3]
```diff
@@ -130,8 +137,10 @@ void PartitionsCacheRepository::Put(int64_t catalogVersion,
 boost::optional<model::LayerVersions> PartitionsCacheRepository::Get(
     int64_t catalogVersion) {
   std::string hrn(hrn_.ToCatalogHRNString());
-  auto cachedLayerVersions = cache_->Get(
-      CreateKey(hrn, catalogVersion), [](const std::string& serializedObject) {
+  auto key = CreateKey(hrn, catalogVersion);
+  LOG_TRACE_F(kLogtag, "Get '%s'", key.c_str());
```
Comment (andescu) [inline olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp]: LOG_INFO_F?

File: olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp (hunk 2 above)
Comment (sergii-vostrikov) [reply olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp]: due to decision to put trace level at the start of a method

File: olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp (hunk 3 above)
Comment (sergii-vostrikov) [reply olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp]: ditto

File: olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp [hunk 4]
```diff
@@ -142,23 +151,23 @@ boost::optional<model::LayerVersions> PartitionsCacheRepository::Get(
 
 void PartitionsCacheRepository::Clear(const std::string& layer_id) {
   std::string hrn(hrn_.ToCatalogHRNString());
+  auto key = CreateKey(hrn, layer_id, boost::none);
+  LOG_TRACE_F(kLogTag, "Clear '%s'", key.c_str());
   cache_->RemoveKeysWithPrefix(hrn + "::" + layer_id);
```
Comment (andescu) [inline olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp]:
```suggestion
  cache_->RemoveKeysWithPrefix(key);
```

Comment (sergii-vostrikov) [reply olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp]: done

**Pull request merged**